src/quickgui/qgsquickidentifykit.h

```diff
@@ -0,0 +1,121 @@
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSQUICKIDENTIFYKIT_H
+#define QGSQUICKIDENTIFYKIT_H
+
+#include <QObject>
+#include <QPair>
+
+#include "qgsfeature.h"
+#include "qgsmapsettings.h"
+#include "qgspoint.h"
+#include "qgsrendercontext.h"
+
+#include "qgis_quick.h"
+#include "qgsquickfeature.h"
+
+class QgsMapLayer;
+class QgsQuickMapSettings;
+class QgsVectorLayer;
+
+/**
+ * \ingroup quick
+ * Convenient set of tools to get a list of QgsFeatures in a defined radius from a point.
+ * Also possible to get a feature with the closest distance to the point or feature(s) from
+ * specified QgsVectorLayer.
+ *
+ * \note QML Type: IdentifyKit
+ *
+ * \since QGIS 3.2
+ */
+class QUICK_EXPORT QgsQuickIdentifyKit : public QObject
+{
+    Q_OBJECT
+
+    /**
+      * Map settings. Set directly when creating QML object.
+      */
+    Q_PROPERTY( QgsQuickMapSettings *mapSettings READ mapSettings WRITE setMapSettings NOTIFY mapSettingsChanged )
+
+    /**
+      * Search radius for the identify functions from the point. Default is 8.
+      */
+    Q_PROPERTY( double searchRadiusMm READ searchRadiusMm WRITE setSearchRadiusMm NOTIFY searchRadiusMmChanged )
+
+    /**
+      * Maximum number of feature returned from by the identify functions in QgsFeatureList. Default is 100.
+      */
+    Q_PROPERTY( long featuresLimit MEMBER mFeaturesLimit NOTIFY featuresLimitChanged )
+
+  public:
+    //! Constructor of new identify kit.
+    explicit QgsQuickIdentifyKit( QObject *parent = 0 );
+
+    //! \copydoc QgsQuickIdentifyKit::mapSettings
+    QgsQuickMapSettings *mapSettings() const;
+
+    //! \copydoc QgsQuickIdentifyKit::mapSettings
+    void setMapSettings( QgsQuickMapSettings *mapSettings );
+
+    //! \copydoc QgsQuickIdentifyKit::searchRadiusMm
+    double searchRadiusMm() const;
+
+    //! \copydoc QgsQuickIdentifyKit::searchRadiusMm
+    void setSearchRadiusMm( double searchRadiusMm );
+
+    /**
+      * Gets the closest feature to the point from the layer in case it is identifiable layer.
+      */
+    Q_INVOKABLE QgsFeature identifyOne( QgsVectorLayer *layer, const QPointF &point );
+
+    /**
+      * Gets the closest feature to the point from any identifiable layer.
+      */
+    Q_INVOKABLE QgsQuickFeature identifyOne( const QPointF &point );
+
+    /**
+      * Gets all features interseting the point from the layer in case it is identifiable layer
+      */
+    Q_INVOKABLE QgsFeatureList identify( QgsVectorLayer *layer, const QPointF &point );
+
+    /**
+      * Gets all features interseting the point from any identifiable layer.
+      */
+    Q_INVOKABLE QList<QgsQuickFeature> identify( const QPointF &point );
+
+
+  signals:
+    //! Map settings changed
```

Comment:
copydoc for all of those, docs only for properties, copydoc for signals and getters/setters